### ISSUE TITLE
update to Occlum NGO

### ISSUE THREAD

**Issue by mythi** [confidential-containers/enclave-cc#112]
- [ ] adapt `shim-rune`to unix-domain-socket
- [ ] update `agent-instance` and `boot-instance` builds to install official Occlum debian packages

**Comment by mythi** [comment]
@qzheng527 what's the latest on Occlum NGO/1.0?

**Comment by mythi** [comment]
> If it is the runtime-boot feature, we can merge it to the Occlum master branch. Is it Ok for you?

@qzheng527 NGO is not urgent. I'm primarily interested in #126 (i.e., get your `enclave-cc` branch rebased to the latest release and have deb packages available just like we have today for the older version)

**Comment by qzheng527** [comment]
> > If it is the runtime-boot feature, we can merge it to the Occlum master branch. Is it Ok for you?
> 
> @qzheng527 NGO is not urgent. I'm primarily interested in #126 (i.e., get your `enclave-cc` branch rebased to the latest release and have deb packages available just like we have today for the older version)

@mythi Ok, I will follow this after International Worker's Day holiday.

**Comment by mythi** [comment]
@qzheng527 does 0.29.6 include the runtime-boot functionality we are using in enclave-cc?

**Comment by qzheng527** [comment]
@mythi Yes, the runtime boot functionality is in this release. The deb package building is WIP, hopefully it is will be done tomorrow (China time).  But please note, the runtime boot functionality in this release is a little bit different from the one you used. Main differences are:
1. The runtime boot struct **user_rootfs_config**. The release version could support passing envs.
2. The way to generate runtime rootfs. The release version uses **"sefs"** instead of the previous **"unionfs"**.

Details please refer to the [runtime_boot](https://github.com/occlum/occlum/tree/master/demos/runtime_boot) demo.

**Comment by mythi** [comment]
@qzheng527 thanks a lot! We will look into the necessary adjustments
